README.md

```diff
@@ -1 +1,60 @@
-# ul-image-api
\ No newline at end of file
+# UL Image API
```

Comment:
Agreed, I will migrate to using a GPII repo before any code is submitted.  The ticket for this is [GPII-2169](https://issues.gpii.net/browse/GPII-2169).